Gemfile

```diff
@@ -98,3 +98,5 @@ gem 'kaminari', '~> 1.2'
 
 # Encrypt attrs
 gem 'attr_encrypted'
+
+gem 'data_migrate'
```

Comment:
The singleton organization is different from seed data because a deployment would be *invalid* without this entry.

I'm OK to have a look into `db:seed` and see if we can build something ourselves. But I would like to treat seeds and static (but required) data differently.

Reply:
> The singleton organization is different from seed data because a deployment would be invalid without this entry.

What do you mean by this. This sounds concerning to me. The "singleton organization" already exists on all production and staging environments and we should most certainly not try to create another organization as part of a deployment.

You also didn't answer my question. Is the purpose of the `data_migrate` gem to replace the pattern that I follow of adding a rake task and an ansible task for data manipulations ie `lib/tasks/temporary/scope_settings_to_organization.rake`?